solutions/CSES Ferris Wheel.mdx

```diff
@@ -2,14 +2,27 @@
 id: cses-1090
 source: CSES
 title: Ferris Wheel
-author: Danh Ta Chi Thanh
+author: Danh Ta Chi Thanh, Kenny Cho
 ---
 
-
 ## Solution
 
 [Unofficial Editorial](https://codeforces.com/blog/entry/83295)
 
+Since each gondola can contain either 1 or 2 children, starting with the heaviest child, we can either
+1. pair the heaviest child with the lightest child if possible; or
+2. only include the heaviest child.
+
+Alternatively, we can start with the lightest child. For each gondola, we pair the lightest child with
+the heaviest child possible without exceeding the weight limit. Those that are left each get their own gondola. 
+
+**Time Complexity:** $\mathcal{O}(n\log n)$ (from sorting)
+
+This solution starts with the lightest child:

```

Comment:
Depending on how you look at this, it *is* starting with the heaviest child (focus on variable $j$).

Reply:
Yeah, that's how the two pointers technique works

Reply:
True. I made edits to try and clarify the explanation.